test/Worker.Extensions.Rpc.Tests/GrpcHttpClientBuilderExtensionsTests.cs

```diff
@@ -21,22 +21,7 @@ public void ConfigureForFunctionsHostGrpc_Configure_SetsUri()
 
         [Fact]
         public void ConfigureForFunctionsHostGrpc_SetsUri()
-        {
-            // https://github.com/grpc/grpc-dotnet/issues/2158
-            // NOTE: when this test starts failing, it means the above issue was fixed.
-            // We should update this test and documentation to reflect as such.
-            InvalidOperationException exception = null;
-            try
-            {
-                ConfigureForFunctionsHostGrpc(s => s.AddGrpcClient<CallInvokerExtractor>());
-            }
-            catch (InvalidOperationException ex)
-            {
-                exception = ex;
-            }
-
-            Assert.NotNull(exception);
-        }
```

Comment:
@jviau -- this appears to be fixed now; i *think* this is the correct fix to the test?